### DISCUSSION TITLE
Hi, where my CHIA! Help pls.

### DISCUSSION THREAD

**Discussion by stro4755** [Chia-Network/chia-blockchain#6795] [Support]
twice I sent CHIA from MXC stock and it was quickly credited to my wallet, 
Yesterday I made the same payment but still no money.
all  screenshots in attached file
[CHIA.docx](https://github.com/Chia-Network/chia-blockchain/files/6660618/CHIA.docx)

**Comment by loppefaaret** [comment]
unfortunately, im not going to download and open a docx document - if you could please upload the images somewhere, and link them instead! or provide the necessary information here please

**Comment by BrandtH22** [comment]
@stro4755 , since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
